pyserini/eval/trec_eval.py

```diff
@@ -18,17 +18,59 @@
 import subprocess
 import sys
 import platform
+import pandas as pd
+import tempfile
 
 from pyserini.search import get_qrels_file
 from pyserini.util import download_evaluation_script
 
 script_path = download_evaluation_script('trec_eval')
 cmd_prefix = ['java', '-jar', script_path]
 args = sys.argv
+# Option to discard non-judged hits in run file
+judged_only = ''
+judgeds = []
+cutoffs = [10, 20]
+if '-judged-only' in args:
```

Comment:
I like more "active" options... how about `-remove-judged`?

Reply:
Allright, but I guess it would be `-remove-unjudged` instead because it only removes non-judged query-document pairs. Do you agree?

Reply:
yea, sorry, my bad.